### DISCUSSION TITLE
Response more than one times for a http request.

### DISCUSSION THREAD

**Discussion by jiajun-c** [tokio-rs/axum#1975] [Q&A]
- [x] I have looked for existing issues (including closed) about this

### Motivation

After handle a request, a continue status code should be send to tell the client to continue request.
![PVYIp24Wox](https://user-images.githubusercontent.com/74438053/235351376-1e13af27-3938-413d-aa05-afbf5918a3af.jpg)

**Comment by davidpdrsn** [comment]
Please ask in `#hyper` in [the tokio discord](https://discord.gg/tokio). I'm not quite sure how to do it and it depends on hyper.

**Comment by jiajun-c** [comment]
hyper add support for 100(status code of continue) in this issue https://github.com/hyperium/hyper/issues/838, just like what I show in the prcture ->client Request ->Server Response 200 or others ->Server Response 100 to continue -> Client continue Request ... Is it possible for axum do like this ?

**Reply by davidpdrsn** [reply]
Yes. Just do that you'd also do for hyper. I have never done it myself so don't know exactly how.
